css/calendarlist.css

```diff
@@ -112,6 +112,9 @@
 #app-navigation .app-navigation-list-item .utils .action span {
 	opacity: .3;
 }
+#app-navigation .app-navigation-list-item .utils .action.withitems span {
+	opacity: .7 !important;
```

Comment:
The `!important` rule overrides the `:hover` selector below, so the icon opacity won't turn to 1 we the mouse hovers it.